tests/network/vmi_infosource.go

```diff
@@ -104,23 +105,26 @@ var _ = SIGDescribe("Infosource", func() {
 
 			expectedInterfaces := []kvirtv1.VirtualMachineInstanceNetworkInterface{
 				{
-					InfoSource: netvmispec.InfoSourceDomain,
-					MAC:        primaryInterfaceMac,
-					Name:       primaryNetwork,
-					QueueCount: network.DefaultInterfaceQueueCount,
+					InfoSource:       netvmispec.InfoSourceDomain,
+					MAC:              primaryInterfaceMac,
+					Name:             primaryNetwork,
+					PodInterfaceName: namescheme.PrimaryPodInterfaceName,
```

Comment:
The test seems to fail becuse `podInterfaceName` is not as expected 

```
the missing elements were
      <[]v1.VirtualMachineInstanceNetworkInterface | len:3, cap:3>: [
          {
              IP: "",
              MAC: "02:00:05:05:05:05",
              Name: "default",
              IPs: nil,
              PodInterfaceName: "eth0",
              InterfaceName: "",
              InfoSource: "domain",
              QueueCount: 1,
          },
          {
              IP: "",
              MAC: "02:00:a0:a0:a0:a0",
              Name: "bridge1-unchanged",
              IPs: nil,
              PodInterfaceName: "pod678b96116fb",
              InterfaceName: "eth1",
              InfoSource: "domain, guest-agent, multus-status",
              QueueCount: 1,
          },
          {
              IP: "",
              MAC: "02:00:a1:a1:a1:a1",
              Name: "bridge2-setns",
              IPs: nil,
              PodInterfaceName: "pod5a30c9d2cef",
              InterfaceName: "",
              InfoSource: "domain, multus-status",
              QueueCount: 1,
          },
      ]
 ```

  ```
  the extra elements were
      <[]v1.VirtualMachineInstanceNetworkInterface | len:3, cap:3>: [
          {
              IP: "",
              MAC: "02:00:05:05:05:05",
              Name: "default",
              IPs: nil,
              PodInterfaceName: "",
              InterfaceName: "",
              InfoSource: "domain",
              QueueCount: 1,
          },
          {
              IP: "",
              MAC: "02:00:a0:a0:a0:a0",
              Name: "bridge1-unchanged",
              IPs: nil,
              PodInterfaceName: "",
              InterfaceName: "eth1",
              InfoSource: "domain, guest-agent, multus-status",
              QueueCount: 1,
          },
          {
              IP: "",
              MAC: "02:00:a1:a1:a1:a1",
              Name: "bridge2-setns",
              IPs: nil,
              PodInterfaceName: "",
              InterfaceName: "",
              InfoSource: "domain, multus-status",
              QueueCount: 1,
          },
      ]

```

Reply:
you need new kubevirtci for new multus (not bumped yet as part of kubevirt)
you can cherry-pick it manually meanwhile in this PR

Reply:
> you need new kubevirtci for new multus (not bumped yet as part of kubevirt) you can cherry-pick it manually meanwhile in this PR

@oshoval , do you see the bug that the new multus version is fixing manifested here?

Reply:
in order to find the default interface, you need to have the default field correct, and this was fixed only on multus 4.1.1 as far as i understand

Reply:
just cherry-pick this one to try
https://github.com/kubevirt/kubevirt/pull/13041
note that it will break other tests of network 

so you can also create a provider that removes calico bump meanwhile if you want to have it clean (temporary of course)

Reply:
> in order to find the default interface, you need to have the default field correct, and this was fixed only on multus 4.1.1 as far as i understand

The theory is not always in sync with reality. I do not see a problem with the mentioned multus issue, it does not seem to be manifested here.
I guess the diff shown is from this run failure: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/13018/pull-kubevirt-e2e-k8s-1.31-sig-network/1844356434607738880
You can double check to confirm it, maybe I did miss something.

Reply:
> this test is failing, either the test is wrong or we have a problem with the population of `podInterfaceName`

I am guessing that the PR is in draft because there are still issues to solve.
Until it is un-draft , we can just try to find the problems and suggest solutions.

Reply:
Hi @oshoval, the failure is not related to the Multus version being used.
It was a failure with the new logic.

Reply:
@orelmisan , please resolve this thread.